ui/lib/package.json

```diff
@@ -18,15 +18,15 @@
     "clean": "rm -rf dist"
   },
   "dependencies": {
-    "@apeworx/apepay": "^0.1.0-alpha.13",
+    "@apeworx/apepay": "^0.1.0-beta.0",
```

Comment:
Should this be beta.1?

Reply:
Hmm, next time should remember to update that after releasing the sdk